Sources/TuistSigning/ProvisioningProfile/ProvisioningProfile.swift

```diff
@@ -0,0 +1,98 @@
+import Foundation
+import TSCBasic
+
+/// Model of a provisioning profile
+struct ProvisioningProfile: Equatable {
+    /// Path to the provisioning profile
+    var path: AbsolutePath?
+    let name: String
+    let targetName: String
+    let configurationName: String
+    let uuid: String
+    let teamId: String
+    let appId: String
+    let appIdName: String
+    let applicationIdPrefix: [String]
+    let platforms: [String]
+    let expirationDate: Date
+
+    init(path: AbsolutePath? = nil,
```

Comment:
Since this is a struct, the internal initialiser is auto-synthesised for you - so you can remove this